src/components/modals/ModalSelectQuestionBank.vue

```diff
@@ -8,12 +8,12 @@
             <span class="modal__close" @click="close"><b-icon icon="times" size="lg"></b-icon></span>
           </div>
           <div class="modal__body">
-            <div class="columns is-multiline" v-if="isLoading">
-              <div class="column is-12" v-for="n in 3" :key="n">
-                <ListItem :minHeight="'50px'" :simple="true" :loading="isLoading"></ListItem>
+            <div v-if="!isLoading" class="columns is-mobile">
+              <div class="column is-flex">
+                <b-checkbox v-model="allSelected" :indeterminate="partialSelected">Select all question bank</b-checkbox>
```

Comment:
`indeterminate` means if `partialSelected` is true, then the v-model will follow `partialSelected` right?